includes/admin/class-wc-admin-menus.php

```diff
@@ -33,7 +33,6 @@ public function __construct() {
 
 		add_action( 'admin_head', array( $this, 'menu_highlight' ) );
 		add_action( 'admin_head', array( $this, 'menu_order_count' ) );
-		add_filter( 'menu_order', array( $this, 'menu_order' ) );
```

Comment:
`menu_order()` can probably be removed now, but I'm not sure about your deprecation policies, so I thought I'd ask for advice first?

Reply:
Our policy is to never remove public methods/functions, even those that aren't actively used anymore within WooCommerce itself, for backwards compatibility with plugins that might be using them. At most we may mark such methods as 'deprecated'.